drivers/rundeck/src/main/kotlin/com/gatehill/corebot/driver/rundeck/action/RundeckJobTriggerService.kt

```diff
@@ -114,4 +115,31 @@ class RundeckJobTriggerService @Inject constructor(private val actionDriver: Run
             }
         })
     }
+
+    override fun fetchExecutionOutput(trigger: TriggerContext, action: ActionConfig, executionId: Int) {
+
+        val call = actionDriver.buildApiClient().fetchExecutionOutput(
+                executionId = executionId.toString()
+        )
+
+        call.enqueue(object : Callback<ExecutionOutput> {
+            override fun onFailure(call: Call<ExecutionOutput>, cause: Throwable) =
+                    handleOutputFailure(trigger, action, executionId, cause)
+
+            override fun onResponse(call: Call<ExecutionOutput>, response: Response<ExecutionOutput>) {
+                if (response.isSuccessful) {
+                    var output: String = ""
+                    for (entry in response.body().entries){
```

Comment:
Could replace this with a `.map` call to make it more functional than imperative.